src/fs/file.rs

```diff
@@ -337,13 +342,23 @@ impl<'dir> File<'dir> {
     }
 
     /// This file’s last accessed timestamp.
+    /// If the file's time is invalid, assume it was accessed today
     pub fn accessed_time(&self) -> Duration {
-        self.metadata.accessed().unwrap().duration_since(UNIX_EPOCH).unwrap()
+        if self.metadata.modified().unwrap() < UNIX_EPOCH{
```

Comment:
Is this a copy paste error converting `accessed` to `modified`?

Reply:
Fixed in 215b779d3584d6bc7fb619ef05274807e7de85f8